### ISSUE TITLE
Declination negative

### ISSUE THREAD

**Issue by blancmatter** [TOMToolkit/tom_lt#4]
Pawel Zielinski submitted an observation using tom_lt on target Gaia19dmj.

The Telescope Observed a target at -23 degrees declination as opposed to +23 degrees.

The log was checked on ltproxy at /usr/local/tomcat/logs/catalina.out and the following RTML document had been sent, clearly showing a -ve degree value. This was also reflected in the Phase2 Gaia19dmj target.

```
2020-05-26 T 17:34:22.245 UTC : org.estar.node_agent2.NodeAgentWebServiceImpl : - : invoked handle_rtml: <RTML xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.rtml.org/v3.1a" mode="request" uid="1590514461" version="3.1a" xsi:schemaLocation="http://www.rtml.org/v3.1a http://telescope.livjm.ac.uk/rtml/RTML-nightly.xsd">
<Project ProjectID="OL20A02">
<Contact>
<Username>XXXXXXXX</Username>
<Name></Name>
</Contact>
</Project>
<Schedule>
<Device name="Sprat" type="spectrograph">
<SpectralRegion>optical</SpectralRegion>
<Setup>
<Grating name="red"/>
<Detector>
<Binning>
<X units="pixels">1</X>
<Y units="pixels">1</Y>
</Binning>
</Detector>
</Setup>
</Device>
<Exposure count="1">
<Value units="seconds">2000.0</Value>
</Exposure>
<Target name="Gaia19dmj">
<Coordinates>
<RightAscension>
<Hours>20</Hours>
<Minutes>5</Minutes>
<Seconds>35.87760000001623</Seconds>
</RightAscension>
<Declination>
<Degrees>-26</Degrees>
<Arcminutes>10</Arcminutes>
<Arcseconds>3.6840000000012196</Arcseconds>
</Declination>
<Equinox>2000.0</Equinox>
</Coordinates>
</Target>
<AirmassConstraint maximum="2.0"/>
<SkyConstraint>
<Flux>4.0</Flux>
<Units>magnitudes/square-arcsecond</Units>
</SkyConstraint>
<SeeingConstraint maximum="2.0" units="arcseconds"/>
<ExtinctionConstraint>
<Clouds>light</Clouds>
</ExtinctionConstraint>
<DateTimeConstraint type="include">
<DateTimeStart system="UT" value="2020-05-27T02:00:00+00:00"/>
<DateTimeEnd system="UT" value="2020-05-27T05:00:00+00:00"/>
</DateTimeConstraint>
</Schedule>
</RTML>
```

**Comment by cylammarco** [comment]
probably from this?
https://github.com/TOMToolkit/tom_lt/blob/169ce570aeba9bf80fd69850d29be33e07a61de9/tom_lt/lt.py#L165

**Comment by blancmatter** [comment]
Yes, the value of `dec.signed-dms.sign` is a numeric value not a string for an astropy coordinate object. 

Now fixed in version 0.4.0
